README.md

```diff
@@ -0,0 +1,40 @@
+# [Archivematica](https://www.archivematica.org/)
+
+By [Artefactual](https://www.artefactual.com/)
+
+Archivematica is a web- and standards-based, open-source application which allows your institution to preserve long-term access to trustworthy, authentic and reliable digital content.
+
+You are free to copy, modify, and distribute Archivematica with attribution under the terms of the AGPL license.
+See the [LICENSE](LICENSE) file for details.
+
+
+## Installation
+
+* [Production installation](https://www.archivematica.org/en/docs/archivematica-1.5/admin-manual/installation/installation/)
```

Comment:
It would be better if this linked to a version-independent or always-latest set of installation instructions.